python/ray/includes/common.pxd

```diff
@@ -167,6 +167,7 @@ cdef extern from "src/ray/protobuf/common.pb.h" nogil:
     cdef CWorkerType WORKER_TYPE_DRIVER "ray::WorkerType::DRIVER"
     cdef CWorkerType WORKER_TYPE_SPILL_WORKER "ray::WorkerType::SPILL_WORKER"
     cdef CWorkerType WORKER_TYPE_RESTORE_WORKER "ray::WorkerType::RESTORE_WORKER"  # noqa: E501
+    cdef CWorkerType WORKER_TYPE_UTIL_WORKER "ray::WorkerType::UTIL_WORKER"  # noqa: E501
```

Comment:
I think these changes are unnecessary?

Reply:
Let me merge to upstream/master. I have another fix for this one which just merged.